woocommerce/google-listings-and-ads
# Streamline campaign setup: When the accounts have been connected before, skip accounts setup step during the Ads-onboarding

### Changes proposed in this Pull Request:

Closes #2534.

_Replace this with a good description of your changes & reasoning._


### Screenshots:

After removal of accounts setup page.
![Screenshot 2024-09-09 at 09 11 26](https://github.com/user-attachments/assets/cba7ffb7-bcbc-4324-a48c-076edbd6ee83)



### Detailed test instructions:


1. Delete `gla_ads_setup_completed_at` options. 
2. Go to dashboard, under `Performance (Paid Campaigns)` section click `Add paid campaign`  
3. You should not see the `Setup accounts page` as shown in the screenshot. Proceed with the other steps and see if you are able to create first paid campaign.


### Additional details:


### Changelog entry

> Update - When the accounts have been connected before, skip accounts setup step during the Ads-onboarding.
